### PR TITLE
SirsiDynix Issues: Emails not sent to organizational contacts

### PR DESCRIPTION
**Pull request by queryluke**
There were 2 problems:

1. `organizationContactsArray()` was not an available method on the Resource class
2. `$email` was not in the foreach loop

### CODE REVIEW

**Comment by t4k** [comment]
Can you lay out a test procedure so we can make sure this is working as expected?

**Comment by queryluke** [comment]
Sure thing.

**Comment by queryluke** [comment]
Sorry, did you mean a manual test procedure (found below) or an automated test?

## REVIEW PROCEDURE
1. Choose or make a resource to test, e.g. __Test Resource 1__
2. In the Organizations module, add a test contact to the vendor of __Test Resource 1__. Make sure the contact has an email address that you can access.
3. In the Resources module, select __Test Resource 1__ > Issues > Report New Issue
4. Select the test contact, add a subject and body
5. Submit the Issue
6. The test contact should receive an email with the subject and body.

**Comment by t4k** [comment]
Manual is great for now! I and others are actively working on getting our automated test framework working and easy to use for everyone. Eventually we’ll want to require tests for all new and fixed functionality.

**Comment by veggiematts** [comment]
Could you resolve the conflicts, please?

**Comment by queryluke** [comment]
@veggiematts done.

File: resources/ajax_processing/insertIssue.php [hunk 1]
```diff
@@ -83,12 +83,12 @@
 		unset($newIssueContact);
 	}
 
-	$organizationContactsArray = $sourceResource->organizationContactsArray($sourceOrganizationID);
+	$organizationContactsArray = $sourceResource->getUnarchivedContacts();
```
Comment (veggiematts) [inline resources/ajax_processing/insertIssue.php]: getUnarchivedContacts is now a ResourceAcquisition method.
Use:
`$organizationContactsArray = $sourceResourceAcquisition->getUnarchivedContacts();`

**Pull request merged**